ui/src/app/views/concept-homepage/component.html

```diff
@@ -0,0 +1,83 @@
+<app-top-box>
+  <clr-tabs>
+    <clr-tab>
+      <button clrTabLink (click)="searching = false;">Concepts</button>
+    </clr-tab>
+    <clr-tab>
+      <button clrTabLink>Concept Sets</button>
+    </clr-tab>
+  </clr-tabs>
+  <div style="display: flex; align-items: center; margin-top: 1.5%;">
+    <clr-icon shape="search" style="position: absolute; height: 1rem; width: 1rem; fill: #216FB4; left: 5.3%;"></clr-icon>
+    <input type="text" id="concept-search-input" [(ngModel)]="searchTerm" placeholder="Search concepts in domains">
+    <span role="tooltip" aria-haspopup="true" class="tooltip tooltip-md tooltip-top-right btn-search-container">
+      <button class="btn btn-secondary btn-search" (click)="searchButton()" [disabled]="getSearchDisabled">
+        Search
+        <span style="padding-left: 0.5rem; display: flex;" *ngIf="searchLoading"><span class="spinner spinner-sm"></span></span>
+      </button>
+      <span *ngIf="searchTermNotLongEnough" class="tooltip-content">Search must be at least three letters long</span>
+    </span>
+    <clr-checkbox class="standard-concepts-checkbox" [(clrChecked)]="standardConceptsOnly">
+      Standard concepts only
+    </clr-checkbox>
+  </div>
+  <div class="domain-box lower-box" *ngIf="!searching else chartBox">
+    <div *ngIf="loadingDomains else domains" class="spinner-container">
+      <span class="spinner spinner-lg"></span>
+    </div>
+    <ng-template #domains>
+      <div class="card item-card" *ngFor="let domain of conceptDomainList">
+        <div class="concept-name">
+          {{domain.name}}
+        </div>
+        <div class="concept-text">
+          <div>
+            <span class="concept-count-text" *ngIf="!standardConceptsOnly else standardCount">{{domain.allConceptCount}}</span>
+            <ng-template #standardCount>
+              <span class="concept-count-text">{{domain.standardConceptCount}} </span>
+            </ng-template>
+            <span class="concept-label-text">concepts</span>
+          </div>
+          in this domain.
+          <div style="margin-top: 0.5rem">
+            <span class="participant-count-text">{{domain.participantCount}}</span>
+            participants in this domain.
+          </div>
+        </div>
+        <div class="footer"><button class="btn btn-link browse-btn" [disabled]="true" (click)="browseDomain(domain)">Browse Domain</button></div>
+      </div>
+    </ng-template>
+  </div>
+  <ng-template #chartBox>
+    <div class="chart-box lower-box">
+      <div class="chart-selector-tabs">
+        <div *ngFor="let domain of conceptDomainList" class="chart-selector-tab">
+          <button (click)="selectDomain(domain.domain)" class="domain-selector-button btn btn-link nav-link nav-item" [class.active]="domain.domain === selectedDomain" [disabled]="searchLoading">
+            <div class="chart-selector-header">
+              {{domain.name}}
+            </div>
+            <div>{{domain.allConceptCount}}</div>
```

Comment:
Does this need  to be based on standrd check box: 
{{ standardConceptsOnly === true ? domain.standardConceptCount : domain.allConceptCount }}

Reply:
This is a placeholder, will add a todo though. (Should be counts on the search, not total counts.

Reply:
That will change the numbers on the cards. These only show up if you have searched anyway